### PR TITLE
ci: enable dailies for nextcloud 29

### LINKED ISSUE #2737: Enable dailies for Nextcloud 29
- [x] [Request `29` track](https://forum.snapcraft.io/t/track-29-for-nextcloud/40101)
- [ ] Setup channel on launchpad
- [x] Merge #2738
- [ ] Merge with next release to `master`

### PR DESCRIPTION
**Pull request by Pilzinsel64**
Fix #2737

Leaving dailies for nextcloud 26 as they still get generated on nextcloud.

### CODE REVIEW

File: .github/spawn-dailies.sh [hunk 1]
```diff
@@ -4,6 +4,7 @@ latest_master_url="https://download.nextcloud.com/server/daily/latest-master.tar
 latest_stable26_url="https://download.nextcloud.com/server/daily/latest-stable26.tar.bz2"
 latest_stable27_url="https://download.nextcloud.com/server/daily/latest-stable27.tar.bz2"
 latest_stable28_url="https://download.nextcloud.com/server/daily/latest-stable28.tar.bz2"
+latest_stable28_url="https://download.nextcloud.com/server/daily/latest-stable29.tar.bz2"
```
Comment (pachulo) [inline .github/spawn-dailies.sh:7]:
```suggestion
latest_stable29_url="https://download.nextcloud.com/server/daily/latest-stable29.tar.bz2"
```

**Review by scubamuc — APPROVED** [review]

**Comment by Pilzinsel64** [comment]
Cleaned git history, so this should be fine now.

**Comment by kyrofa** [comment]
> Leaving dailies for nextcloud 26 as they still get generated on nextcloud.

Only [29, 28, and 27](https://nextcloud.com/changelog/) are actually supported. Time to remove 26, I think. We need to update our [release strategy](https://github.com/nextcloud-snap/nextcloud-snap/wiki/Release-strategy) page as well to reflect the current state of things.

**Comment by Pilzinsel64** [comment]
> Time to remove 26,

Yes Sir. Done!

(When creating this commit I just throught "why not, as long upstream is active" in first point. And it's simpler to add a commit instead of removing one. 🙃)

**Comment by Pilzinsel64** [comment]
Rebased. Would be cool to get this in before we do a new release with nc28.

File: .github/spawn-dailies.sh [hunk 2]
```diff
@@ -4,6 +4,7 @@ latest_master_url="https://download.nextcloud.com/server/daily/latest-master.tar
 latest_stable26_url="https://download.nextcloud.com/server/daily/latest-stable26.tar.bz2"
```
Comment (kyrofa) [inline .github/spawn-dailies.sh:4]: Remove this as well.

**Review by kyrofa — APPROVED** [review]
Small nit, but +1 overall.

File: .github/spawn-dailies.sh (hunk 2 above)
Comment (Pilzinsel64) [reply .github/spawn-dailies.sh]: Oh thanks, good catch. Just forgot about this line. 😄

**Pull request merged**